laborator/content/introducere-asamblare/README.md

```diff
@@ -256,6 +256,10 @@ Calculați minimul dintre numerele din 2 registre (eax și ebx) folosind o instr
 
 Calculați al N-lea număr Fibonacci, unde N este dat prin intermediul registrului `eax`.
 
+### 6. BONUS: Fibonacci
+
+Calculați al N-lea număr Fibonacci, unde N este dat prin intermediul registrului `eax`.
+
```

Comment:
There is something strange here. Commit subject says you add CF vs OF example, but here you also add another  Fibonacci exercise. 

This is confusing. A commit should only do one thing.